lib/main.dart

```diff
@@ -98,16 +97,16 @@ class _MyHomePageState extends State<MyHomePage> {
 }
 
 class SectionHeader extends StatelessWidget {
-  SectionHeader({Key key, this.title}) : super(key: key);
+  SectionHeader({Key? key, this.title}) : super(key: key);
 
-  final String title;
+  final String? title;
 
   @override
   Widget build(BuildContext context) {
     return Container(
       padding: EdgeInsets.all(10.0),
       child: Text(
-        title,
+        title!,
```

Comment:
if we are going to unconditionally null-assert this, let's make `title` non nullable